frontend/src/pages/study-room-page/components/side-menu/SideMenu.tsx

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/react';
+
+import type { TabId, Tabs } from '@study-room-page/StudyRoomPage';
+import * as S from '@study-room-page/components/side-menu/SideMenu.style';
+import TabButton from '@study-room-page/components/tab-button/TabButton';
+
+export interface SideMenuProps {
+  className?: string;
+  activeTabId: TabId;
+  tabs: Tabs;
+  handleTabButtonClick: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+const mb20 = css`
+  margin-bottom: 12px;
+`;
+
+const SideMenu: React.FC<SideMenuProps> = ({ className, activeTabId, tabs, handleTabButtonClick }) => {
+  return (
+    <S.Nav className={className}>
```

Comment:
```suggestion
    <S.Sidebar className={className}>
```
이거는 어떤가요?  
나중에 다른 style파일에서 Sidebar를 불러와서 `${Sidebar} { display: none; }` 이런식으로 쓴다면  
Nav보다는 Sidebar가 더 명시적일것 같아요 :D